lib/_emerge/JobStatusDisplay.py

```diff
@@ -262,6 +263,13 @@ def _display_status(self):
             f.pop_style()
             f.add_literal_data(" failed")
 
+        if self.merge_wait:
+            f.add_literal_data(", ")
+            f.push_style(number_style)
+            f.add_literal_data(f"{self.merge_wait}")
+            f.pop_style()
+            f.add_literal_data(" merge wait")
+
```

Comment:
I guess we could adjust it somewhere to add a space between `merge wait` and `Load Avg` here:
```console
>>> Installing (131 of 259) dev-perl/Crypt-PasswdMD5-1.420.0::gentoo
>>> Jobs: 128 of 259 complete, 5 running            Load avg: 3.06, 2.47, 1.48
>>> Jobs: 128 of 259 complete, 4 running            Load avg: 3.06, 2.47, 1.48
>>> Jobs: 128 of 259 complete, 3 running            Load avg: 3.06, 2.47, 1.48
>>> Installing (133 of 259) dev-perl/Digest-SHA1-2.130.0-r3::gentoo
>>> Jobs: 128 of 259 complete, 3 running, 1 merge waitLoad avg: 3.38, 2.55, 1.51
>>> Jobs: 128 of 259 complete, 2 running, 1 merge waitLoad avg: 3.38, 2.55, 1.51
```

Reply:
I don't think this is necessary here. 

The problem was that https://github.com/gentoo/portage/pull/1348 did not work as intended. Fixed in https://github.com/gentoo/portage/pull/1377